docs/configuration.rst

```diff
@@ -347,3 +347,9 @@ below:
      - Sensitive parameter value regexp for obfuscation.
 
 .. _Unified Service Tagging: https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging/
+
+
+Dynamic instrumentation configuration
```

Comment:
The h1 here is already "Configuration".

```suggestion
Dynamic instrumentation
```

Also, should this instead be "Debugger" to keep with how this feature is talked about in the public interface?